L1Trigger/L1THGCal/test/testHGCalBackendStage1ParameterExtractor_cfg.py

```diff
@@ -59,18 +58,22 @@
 # ID of tested FPGA
 fpga_id = 1
 
+# geometry version
+from Geometry.CMSCommonData.cmsExtendedGeometry2026D49XML_cfi import XMLIdealGeometryESSource
```

Comment:
Should not be retrieved like that (hardcoded), but rather through the `process` object; this way it will automatically extract the geometry depending on the module that has been loaded.
Currently since the import is hardcoded, it is strictly equivalent as writing `geometry_version = 'v11'`

Reply:
Indeed that was silly, now retrieved through `process`